src/Utilities/Utilities.jl

```diff
@@ -300,6 +300,9 @@ doccat(b::Binding, ::Type)  = "Method"
 
 doccat(::Function) = "Function"
 doccat(::DataType) = "Type"
+if VERSION >= v"0.6.0-dev.2122"
```

Comment:
Not sure actually which is better practice, but `isdefined(Base, :UnionAll)` should work here as well, right?